### PR TITLE
Nightwatch Fixes & Travis Integration

### PR DESCRIPTION
**Pull request by awatson1978**
Testing the Github/Travis integration.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  # setup Meteor app
+  - "cd leaderboard-nightwatch"
+    # Optionally we can update all our dependencies here
+    #- "meteor update"
+  - "meteor reset"
+  - "meteor -p 3000 &"
+  # give Meteor some time to download packages, init data, and to start
+  - "sleep 50"
+
+script: ./run_nightwatch.sh
```
Comment (queso) [inline .travis.yml]: So this sits in the root of the app and will need to have each framework added to it then, @awatson1978?

Comment (awatson1978) [inline .travis.yml]: Yes, indeed.  :)

**Comment by awatson1978** [comment]
Aha!  Well, look at that!  :)

**Comment by queso** [comment]
Yay!

:+1: from me.

**Comment by samhatoum** [comment]
!!!

**Comment by samhatoum** [comment]
Nice work @awatson1978!

**Comment by awatson1978** [comment]
Thanks!  And a shout-out to @zeroasterisk for all the work he's done on getting Travis integration working!

**Comment by queso** [comment]
@awatson1978 are you ok if we refactor the setup into a setup script in the nightwatch folder? I think the travis.yml will get kind of messy if we have to add steps like this for each framework

**Comment by awatson1978** [comment]
Yeah, I already started on that with the last commit.  The script running from the root directory gets things running green for the ``velocity-examples`` repository.  We need _something_ there.  But the ``leaderboard-example`` needs it's own ``.travis.yml`` file, so people can see how it's run stand-alone.  

I'd recommend either a) keeping the root ``.travis.yml`` file for either meta-tests, or b) to write it so that it calls scripts from within each of the examples.

**Comment by awatson1978** [comment]
The next pull-request I'm working on will be a meta-script to test the HTML reporter itself as it runs, which is probably more appropriate to be run from the root ``.travis.yml`` file.  That and documentation.

**Comment by queso** [comment]
I'm more a fan of option B. I would assume when end up with a series of scripts that run.

**Comment by awatson1978** [comment]
Works for me.  That's why I added the ``.travis.yml`` file in both places... so that others can hack on the root file.  We'll probably want to change ``/.travis.yml`` from this:

````sh
script: ./run_nightwatch.sh
````

to something like this?

````
script:  
 # cucumber
 - leaderboard-cucumber/some_cucumber_script.sh

 # jasmine
 - leaderboard-jasmine/some_jasmine_script.sh

 # nightwatch
 - leaderboard-nightwatch/run_nightwatch.sh
````

**Pull request merged**